src/request.rs

```diff
@@ -121,7 +121,7 @@ impl<State> Request<State> {
     ///
     /// let mut app = tide::new();
     /// app.at("/").get(|req: Request<()>| async move {
-    ///     assert_eq!(req.header("X-Forwarded-For").unwrap().last().as_str(), "127.0.0.1");
+    ///     assert_eq!(req.header("X-Forwarded-For").unwrap(), "127.0.0.1");
```

Comment:
Shouldn't this be using `req["X-Forwarded-For"]`?

Reply:
I went to make that change a few times, but it's actually the docs for the Request::header method, so it wouldn't make sense not to use Request::header